### DISCUSSION TITLE
how to estimate inference time?

### DISCUSSION THREAD

**Discussion by kimnamu** [open-mmlab/mmdetection#6919] [Q&A] [answered]
As you know, inference time depends on the hardware performance.
But in the inference time(fps) table, there is no hardware specification.

For example, the fps of retinanet fp16 model is 31.6 fps, but it is not easy to achieve a similar speed.
Can I guess the same hardware on the same table?  So could you tell me which GPU is your standard?

**Comment by baibaidj** [comment, marked as answer]
I guess you could find the hardware info in the training log, which was listed in the rightest column of a performance table. See here for instance: https://download.openmmlab.com/mmdetection/v2.0/fp16/retinanet_r50_fpn_fp16_1x_coco/retinanet_r50_fpn_fp16_1x_coco_20200702_020127.log.json

**Reply by kimnamu** [reply]
It's right. Thanks!
